uportal-portlets-overlay/CalendarPortlet/pom.xml

```diff
@@ -49,9 +49,9 @@
         </dependency>
 
         <dependency>
-            <groupId>javax.portlet</groupId>
-            <artifactId>portlet-api</artifactId>
-            <version>2.0</version>
+            <groupId>org.apache.portals</groupId>
+            <artifactId>portlet-api_2.1.0_spec</artifactId>
+            <version>1.0</version>
```

Comment:
Announcements does have a dependency on the artifact. Calendar did not, so removing that reference.